Thanks @patrickimran, looks like there's only one failure:

```
    def testValidJobArrayID(self):
1419        # Since the metadata provided is not SampleData[Target],
1420        # we expect q2_mlab to throw an error with return code 1.
1421        # The downside to this over checking for a timeout is that waiting
1422        # for 20 (n_chunks) errors takes time.
1423    
1424        os.environ["PBS_ARRAYID"] = "5"
1425        completed_process = subprocess.run(
1426            f"{self.test_script}",
1427            shell=True,
1428            stdout=subprocess.PIPE,
1429            stderr=subprocess.PIPE,
1430        )
1431        print(completed_process)
1432>       self.assertEqual(completed_process.returncode, 1)
1433E       AssertionError: 2 != 1
```

--- a/q2_mlab/templates/array_job_template.sh
+++ b/q2_mlab/templates/array_job_template.sh
@@ -86,6 +86,7 @@ do
 
     # Only skip execution if results exist and we aren't forcing it.
     if [[ ${RESULTS_EXIST} = true && ${FORCE} = false ]]
+    then
         echo $RESULTS already exists, execution skipped
     else
         qiime mlab unit-benchmark --i-table {{ TABLE_FP }} \

